client/src/pages/Connect/Connect.css

```diff
@@ -0,0 +1,35 @@
+.get-started-banner {
+    width: 100vw;
+    height: 36vw;
+}
+
+#get-started-body {
+    display: flex;
+}
+
+#get-started-body-mobile {
+    display: block;
+}
+
+.get-started-sub {
+    margin-top: 5vh;
+    width:35vw;
+}
+
+#get-started-form {
+    margin-right: 5vw;
+}
+
+#get-started-form-mobile {
+    width: 80vw;
+}
+
```

Comment:
I'm not sure where exactly, but @chauandrew had a way of making the horizontal scroll bar not just disappear, but no longer necessary?

Reply:
Hmm yeah, it's something that confused me as well. Lemme see if I can do anything about this, probs after waking up tho